README.md

```diff
@@ -1,591 +1,51 @@
-# Giskard
+<p align="center">
+  <img width="300" alt="giskardlogo" src="frontend/src/assets/logo_full.svg">
+</p>
 
-## Backend Requirements
+<h3 align="center">Quality Assurance for AI</h3>
+<br />
+<p align="center">
+    <img src="readme/inspect.png" alt="Administration panel" />
+</p>
 
-* [Docker](https://www.docker.com/)
-* [Docker Compose](https://docs.docker.com/compose/install/)
-* [Poetry](https://python-poetry.org/) for Python package and environment management
+<br/>
 
-## Frontend Requirements
+Giskard is an open-source quality assurance platform for AI models.
```

Comment:
Perhaps this block would be better positioned above the screenshot?

Reply:
agree